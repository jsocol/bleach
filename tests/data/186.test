<textarea type="text" cols="50" rows="10"></textarea>//["'`-->]]>]</div>
