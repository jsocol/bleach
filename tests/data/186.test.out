<textarea rows="10" cols="50" type="text"></textarea>//["'`--&gt;]]&gt;]
